### ISSUE TITLE
Some ideas / feedback

### ISSUE THREAD

**Issue by jorisvandenbossche** [glemaitre/pyparis-2018-sklearn#3]
- [x] what is stratification?
- [x] more help in exercise (pointers to what to use, hidden tip?), or chunked solutions
- [x] MinMaxScaler instead of StandardScaler? -> update text about this
- [x] number of iterations -> remove hard coded number (I get different results)
- [x] pipeline: maybe first show the usage of it (see, we again use fit / score, just like before, but now under the hood ...), and only then explain the inner details with showing get_params
- [x] "The two former functions are a specific case of the latter one. Thus, we will focus on the latter." -> not fully true for `cross_val_predict` ? 
  Also, I would provide some more explanation on `cross_validate`
- [x] " fine-tune the inside of pipeline" -> a word is missing
- [x] cv=5 instead of 3 ?
- [x] Inner parameters -> hyper parameters ? (the "inner parameters" term in not used elsewhere)
- [x] Why nested cross-validation ? -> provide some more context
- [x] Show `pd.DataFrame(grid.cv_results_)` ? -> because I might want to know how much difference there is in the score eg between hinge and log loss
- [x] SimpleImputer -> `fill_value='NA'` is not really needed, as the default of 'missing_value' is already fine
- [x] " For the missing data, we will use the string NA for the categorical columns and np.nan for the numerical columns" -> mixing two concepts
- [x] use `make_column_transformer` instead of `ColumnTransformer` to avoid the possible ordering deprecations ?
